srcpkgs/wlroots/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'wlroots'
+pkgname=wlroots
+version=0.0.1
+revision=1
+_commit=842368ec983cdf671cf16f5be51633392ed52d5e
+wrksrc=$pkgname-$_commit
+build_style=meson
+configure_args="-Dlogind-provider=elogind -Drootston=false
+ -Dexamples=false"
+hostmakedepends="pkg-config wayland-devel wayland-protocols"
+makedepends="wayland-devel wayland-protocols MesaLib-devel
+ libinput-devel libxkbcommon-devel eudev-libudev-devel pixman-devel
+ $(vopt_if libcap libcap-devel) $(vopt_if logind elogind-devel)
+ $(vopt_if x11 'libxcb-devel xcb-util-image-devel libX11-devel
+  xcb-util-errors-devel xcb-util-wm-devel')"
+short_desc="Wayland compositor library"
+maintainer="Emerson Ferreira <me@skrps.me>"
+license="MIT"
+homepage="https://github.com/swaywm/$pkgname"
+distfiles="$homepage/archive/$_commit.tar.gz"
+checksum=4c664fa436a0e313decb5e769c1a86e4db075ddfe7c678dac4844f8d6f6604d2
+
+build_options="logind libcap x11"
+build_options_default="logind libcap x11"
+desc_option_logind="Enable support for rootless session via elogind"
+desc_option_libcap="Enable support for rootless session via capabilities"
+
+post_install() {
+	vlicense LICENSE
+}
+
+wlroots-devel_package() {
+	short_desc+=" - development files "
+	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
```

Comment:
should problably have the .so files if there are any

Reply:
wlroots is a library

Reply:
add vmove "usr/lib/*.so"

Reply:
Oh, the versioned one... silly me :expressionless: